agent/util-scripts/pbench-tool-meister-start

```diff
@@ -964,10 +964,7 @@ def main(_prog: str, cli_params: Namespace) -> int:
     # drive the following client operations ("sysinfo" [optional] and "init"
     # [required]).
     with Client(
-        redis_server=redis_client,
-        channel_prefix=cli_tm_channel_prefix,
-        to_client_chan=to_client_chan,
-        logger=logger,
+        redis_host=redis_server.host, redis_port=redis_server.port, logger=logger
```

Comment:
state-signals currently only uses redis host and port for connection, so that information would have to be extracted at some point (either in pbench-tool-meister-start or in within the Client object creation code). This could be changed in the future with a state-signals update if desired though.

Reply:
I am working on a change to state-signals independent of this to add support for the redis client.